wget/README.md

```diff
@@ -0,0 +1,51 @@
+
+# `wget`
+
+This task performs wget operation to download file from internet to workspace .
+
+## Install the Task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/v1beta1/wget/wget.yaml
+```
+
+## Parameters
+
+* **mwendler/wget**: `wget` CLI container image to run this task.
+
+* **options**: The arguments to pass to `wget` CLI.  _default_: `[""]`
+* **diroptions**: The directory path we want to save file to.  _default_: `[""]`
+* **url**: The url we want to download file from.  _default_: `""`
+* **filename**: The filename we want to change our file to.  _default_: `""`
+## Usage
+
+### Running the Task
+
+#### Download file to the workspace
+
+You can invoke `wget url -P workspace` to check if file is downloaded to workspace:
+
+Create a file, `wget-run.yaml`:
+
+```
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: wget-run
+spec:
+  params:
+    - name: url
+      value: "www.google.com"
+    - name: diroptions
+      value:
+        - "-P"
+  taskRef:
+    name: wget
+  workspaces:
+    - name: myworkspace
+      emptyDir: {}
+```
+
+Run it with `kubectl create -f wget-run.yaml`
```

Comment:
nit: the taskrun uses name but the kubectl command usage uses create. We should either switch the taskrun to use `generateName` or use `kubectl apply`